src/main/webapp/user-page.html

```diff
@@ -38,18 +40,47 @@ <h1 id="page-title">User Page</h1>
         <br/>
         <input type="submit" value="Submit">
       </form>
+    <script>
+      CKEDITOR.replace('about-me');
+    </script>
     </div>
-
+    
     <form id="message-form" action="/messages" method="POST" class="hidden">
       Enter a new message:
       <br/>
       <textarea name="text" id="message-input"></textarea>
       <br/>
       <input type="submit" value="Submit">
     </form>
+    <script>
+      CKEDITOR.replace('text');
+    </script>
     <hr/>
 
     <div id="message-container">Loading...</div>
-
-  </body>
-</html>
\ No newline at end of file
+    <!--
+    <script>
```

Comment:
Please remove commented out code. Only check in code that will be in for production.